plyer/platforms/linux/wifi.py

```diff
@@ -16,8 +16,31 @@
 
 
 class LinuxWifi(Wifi):
+    def __init__(self):
+        self.ifname = self.getIfname()
+        self.nmcliVersion = self.getNmcliVersion()
+
     names = {}
 
+    def getIfname(self):
+        '''
+        Return wifi interface name if available else return "wlan0'.
+        '''
+        com = Popen(["nmcli", "-t", "device"], stdout=PIPE)
+        com = com.communicate()[0].decode().split("\n")
+        for i in com:
+            if("wifi" in i):
```

Comment:
Python does not require brackets for conditions and it decreases the readability. Brackets in conditions are mostly used to explicitly mark an order of some *logical* operations for example:

```python
>>> 1 and (1 or 1) and 0
0
>>> 1 and 1 or 1 and 0
1
>>>
```